unittest/gunit/table_cache-t.cc

```diff
@@ -861,19 +853,11 @@ TEST_F(TableCacheDoubleCacheDeathTest, ManagerFreeTable)
   // There should be assert failure since we are trying
   // to free all not own TABLEs for share_1, while thd_2
   // has a TABLE object for it in used
-<<<<<<< HEAD
-#ifndef DBUG_OFF
-  MY_EXPECT_DEATH_IF_SUPPORTED(table_cache_manager.free_table(thd_1,
-||||||| e5d189ecb94
-#ifndef DBUG_OFF
-  EXPECT_DEATH_IF_SUPPORTED(table_cache_manager.free_table(thd_1,
-=======
 #ifndef NDEBUG
-  EXPECT_DEATH_IF_SUPPORTED(table_cache_manager.free_table(thd_1,
->>>>>>> 37b047220a907c2a6d7235ddf2b7a6be916cc82e
-                                                           TDC_RT_REMOVE_NOT_OWN,
-                                                           &share_1),
-                            ".*Assertion.*0.*");
+  MY_EXPECT_DEATH_IF_SUPPORTED(table_cache_manager.free_table(thd_1,
```

Comment:
Likewise

Reply:
Please, I have included the result of this test in the output above.